.circleci/config.yml

```diff
@@ -106,6 +106,29 @@ commands:
       - store_artifacts:
           path: test/logs
 
+  dockerhub-build-steps:
+    parameters:
+      build_args:
+        type: string
+    steps:
+      - setup-executor
+      - checkout
+      - run:
+          name: Setup
+          command: |
+            git submodule update --init --recursive
+            ./opt/readies/bin/getpy3
+      - run:
+          name: Build for platform
+          command: |
+            #@@ docker login -u redisai -p $DOCKER_REDISAI_PWD
+            cd opt/build/docker
+            #@@ make build publish $(./version-params) CPU=1 OSNICK=bionic X64=1 REPO=redisai VERBOSE=1 <<parameters.build_args>>
+            #@@ make build publish $(./version-params) GPU=1 OSNICK=bionic X64=1 REPO=redisai VERBOSE=1 <<parameters.build_args>>
```

Comment:
comment?

Reply:
Don't yet have password for `dockerhub/redisai` so publish will fail.